<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheHanger</class>
 <widget class="QMainWindow" name="TheHanger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>351</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>591</width>
    <height>351</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>591</width>
    <height>351</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TheHanger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="windowHome" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>351</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>261</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="nameUserLineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="listUsersComboBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonPlay">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter an username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select an username</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>261</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Player :</string>
      </property>
     </widget>
     <widget class="QLabel" name="playerName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Unknow</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Last score :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lastScore">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>unknow</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of parts :</string>
      </property>
     </widget>
     <widget class="QLabel" name="numberOfParts">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>unknow</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Date :</string>
      </property>
     </widget>
     <widget class="QLabel" name="date">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="windowHanger" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>351</height>
     </rect>
    </property>
    <widget class="QLabel" name="username">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
    <widget class="QWidget" name="windowPlay" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>481</width>
       <height>351</height>
      </rect>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="starWord">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>************</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>221</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="numberImg">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>numberImg</string>
       </property>
      </widget>
      <widget class="QLabel" name="img1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>img1</string>
       </property>
      </widget>
      <widget class="QLabel" name="img2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>img2</string>
       </property>
      </widget>
      <widget class="QLabel" name="img3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>img3</string>
       </property>
      </widget>
      <widget class="QLabel" name="img4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>img4</string>
       </property>
      </widget>
      <widget class="QLabel" name="img5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>img5</string>
       </property>
      </widget>
      <widget class="QLabel" name="img6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>img6</string>
       </property>
      </widget>
      <widget class="QLabel" name="img7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>img7</string>
       </property>
      </widget>
      <widget class="QLabel" name="img8">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>img8</string>
       </property>
      </widget>
      <widget class="QLabel" name="img9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>img9</string>
       </property>
      </widget>
      <widget class="QLabel" name="img10">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>img10</string>
       </property>
      </widget>
      <widget class="QLabel" name="img11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>img11</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>241</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of strokes :</string>
       </property>
      </widget>
      <widget class="QLabel" name="numberOfStrokes">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Written letter :</string>
       </property>
      </widget>
      <widget class="QLabel" name="writtenLetters">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>a z e r t y</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Score :</string>
       </property>
      </widget>
      <widget class="QLabel" name="numberScore">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
      <widget class="QLabel" name="secretWord">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>secretWord</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>200</y>
        <width>471</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="keyA">
         <property name="text">
          <string>a</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="keyB">
         <property name="text">
          <string>b</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="keyC">
         <property name="text">
          <string>c</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="keyD">
         <property name="text">
          <string>d</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="keyE">
         <property name="text">
          <string>e</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="keyH">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>h</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="keyI">
         <property name="text">
          <string>i</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="keyJ">
         <property name="text">
          <string>j</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="keyL">
         <property name="text">
          <string>l</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="keyM">
         <property name="text">
          <string>m</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="keyO">
         <property name="text">
          <string>o</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="keyP">
         <property name="text">
          <string>p</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="keyQ">
         <property name="text">
          <string>q</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="keyR">
         <property name="text">
          <string>r</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="keyS">
         <property name="text">
          <string>s</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QPushButton" name="keyT">
         <property name="text">
          <string>t</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QPushButton" name="keyU">
         <property name="text">
          <string>u</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QPushButton" name="keyN">
         <property name="text">
          <string>n</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="keyF">
         <property name="text">
          <string>f</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QPushButton" name="keyZ">
         <property name="text">
          <string>z</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="keyY">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="keyX">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="keyW">
         <property name="text">
          <string>w</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="keyV">
         <property name="text">
          <string>v</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="keyG">
         <property name="text">
          <string>g</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="keyK">
         <property name="text">
          <string>k</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonReplay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Replay</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBack">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
